steps/server02.js

```diff
@@ -0,0 +1,5 @@
+// Create a PORT variable and assign it to 3000, this will be the port your server will listen to
```

Comment:
```
// Create a `PORT` variable and set it equal to 3000. This will be the port your server listens to.
// Call the app's `listen` method by passing it `PORT` as an argument.
```